src/components/seplanen/sendtilgodkjenning/GodkjennEgenPlanAGForm.tsx

```diff
@@ -0,0 +1,169 @@
+import {
```

Comment:
Hva er forskjellen mellom denne og den vanlige `SendTilGodkjenningForm`?

Reply:
Een nye formen har ikke tvangsgodkjenning/send til godkjenning radios

Reply:
Det blir en del duplisering mellom forms nå, kunne kan ha gjenbrukt SendTilGodkjenningForm ved å endre props til for eksempel:

`interface Props {
  oppfolgingsplan: Oppfolgingsplan;
  visTvungenGodkjenningToggle: boolean;
  sendTilGodkjenning: (values: SendTilGodkjenningFormValues) => void;
  isSubmitting: boolean;

  cancel(): void;
}`

Da kan du sende inn sendTilGodkjenning-funksjon som håndterer mapping og sending utenfor formen.